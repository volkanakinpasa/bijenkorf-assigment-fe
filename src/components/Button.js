function Button(props) {
  const { label, icon, type, onClick } = props;

  return (
    <button type={type} aria-label={label} onClick={onClick}>
      {icon}
    </button>
  );
}

export default Button;
