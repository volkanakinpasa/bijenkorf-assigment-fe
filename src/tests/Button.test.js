import Button from '../components/Button';
import { render } from '@testing-library/react';

describe('Button tests', () => {
  it('creates a snapshot', () => {
    const { asFragment } = render(<Button />);
    expect(asFragment()).toMatchSnapshot();
  });

  //add to cover type and onclick
});
