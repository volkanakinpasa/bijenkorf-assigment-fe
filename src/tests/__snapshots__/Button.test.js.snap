// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button tests creates a snapshot 1`] = `
<DocumentFragment>
  <button />
</DocumentFragment>
`;
